src/System.Device.Gpio/Interop/Unix/SafeLineHandle.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Device.Gpio
+{
+    internal class SafeLineHandle : SafeHandle
```

Comment:
question: Can we think of a Chip to be equivalent to a GpioController and a Line to be a Pin? Is that more or less what this maps to? It would be great to have that info in a comment above the class just for future reference

Reply:
Not sure what level of equivalency you mean, yes SafeChipHandle is maps to gpio controller and SafeLineHandle is map to pin, but at hardware level, just pointers to chip/pin. But cannot be equivalent to our Software level GpioController